packages/block-editor/src/components/block-tools/block-selection-button.js

```diff
@@ -241,20 +252,25 @@ function BlockSelectionButton( { clientId, rootClientId } ) {
 					<BlockIcon icon={ blockInformation?.icon } showColors />
 				</FlexItem>
 				<FlexItem>
-					<BlockDraggable clientIds={ [ clientId ] }>
-						{ ( draggableProps ) => (
-							<Button
-								icon={ dragHandle }
-								className="block-selection-button_drag-handle"
-								aria-hidden="true"
-								label={ dragHandleLabel }
-								// Should not be able to tab to drag handle as this
-								// button can only be used with a pointer device.
-								tabIndex="-1"
-								{ ...draggableProps }
-							/>
-						) }
-					</BlockDraggable>
+					{ editorMode === 'exploded' && (
```

Comment:
I think select mode and exploded view should align on the toolbar presentation and control (dark display, drag handle anywhere, use of overlay). Maybe the exploded view introduces the ability to swap patterns, but that's it in terms of toolbar updates.

Reply:
The black makes more sense to me

Reply:
The contrast of black button on very dark grey background didn't seem ideal. But now we're headed in a different direction I agree we should stick to black.

Reply:
Made some tweaks:

 - The button is now black
 - I've unified the "overlay" behavior with the navigation mode as well

Reply:
@jameskoster right, we can always revisit those materials and tweak them, but it makes sense to not introduce a diversion here.